includes/class-edd-license-handler.php

```diff
@@ -156,11 +156,20 @@ public function auto_updater() {
 			return;
 		}
 
+		$license = $this->license;
+		// Fall back to the highest license key if one is not saved for this extension.
+		if ( empty( $license ) ) {
+			$pass_manager = new \EDD\Admin\Pass_Manager();
+			$highest_key  = $pass_manager->highest_license_key;
+			if ( $highest_key ) {
+				$license = $highest_key;
+			}
```

Comment:
```suggestion
			if ( $pass_manager->highest_license_key ) {
				$license = $pass_manager->highest_license_key;
			}
```

The new variable isn't needed.